static/js/compose_actions.js

```diff
@@ -484,7 +490,17 @@ export function quote_and_reply(opts) {
         compose_ui.autosize_textarea($("#compose-textarea"));
     }
 
-    if (message && message.raw_content) {
+    if (message && opts.selected_text !== undefined) {
+        // The copied text is plain text, therefore, the quoted text
+        // is not stylized as well.
+        message.raw_content = opts.selected_text;
+        replace_content(message);
```

Comment:
We should definitely not be doing the hacky `message.raw_content` logic here -- refactor the `replace_content` function to support a way to pass the raw content to use as a second parameter.